Billing.API.Test/InvoiceApiTest.cs

```
@@ -188,5 +188,75 @@ public async Task GetInvoices_WhenNoParameters_ReturnsNotFound()
             // Assert
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
+
+        [Fact]
+        public async Task GetInvoice_ReturnsFile()
+        {
+            // Arrange
+            using var appFactory = _factory.WithBypassAuthorization()
+                .AddConfiguration(new Dictionary<string, string>
+                {
+                    ["Invoice:UseDummyData"] = "true"
+                });
+
+            appFactory.Server.PreserveExecutionContext = true;
+            var client = appFactory.CreateClient();
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"https://custom.domain.com/accounts/doppler/1/invoice/filename.ext");
```

It should throw a 400 or 404 status code, I am missing something?

---

I rewrote this test, was outdated, I added the correct response code, validated that the content type is PDF and also that the responde has contents